var/spack/repos/builtin/packages/py-pystan/package.py

```diff
@@ -14,13 +15,23 @@ class PyPystan(PythonPackage):
 
     maintainers = ['haralmha']
 
-    version('3.4.0', sha256='325e2fb0ab804555c05a603e0c9152ab11fcc3af01f3e9a9ff9fe9954b93184f')
+    version('3.5.0',    sha256='078571d071a5b7c0af59206d4994a0979f4ac4b61f4a720b640c44fe35514929')
+    version('3.4.0',    sha256='325e2fb0ab804555c05a603e0c9152ab11fcc3af01f3e9a9ff9fe9954b93184f')
+    version('2.19.1.1', sha256='fa8bad8dbc0da22bbe6f36af56c9abbfcf10f92df8ce627d59a36bd8d25eb038')
+    version('2.19.0.0', sha256='b85301b960d5991918b40bd64a4e9321813657a9fc028e0f39edce7220a309eb')
 
-    depends_on('python@3.8:3', type=('build', 'run'))
-    depends_on('py-setuptools', type=('build', 'run'))
+    # common requirements
+    depends_on('py-setuptools',         type='build')
```

Comment:
This should probably be type=build/run

Reply:
Should be type=build/run

Reply:
oh, I misread this comment as for py-plotnine.